components/compliance-service/reporting/relaxting/reports.go

```diff
@@ -98,7 +71,7 @@ func (backend ES2Backend) getDocIdHits(esIndex string,
 
 func (backend ES2Backend) getNodeReportIdsFromTimeseries(esIndex string,
```

Comment:
There is a lot happening in the diff for this function, but it had two changes here:
* Changing it's return type from a map to a slice, because that's how the two callers used the data.
* Flipping the condition to return early if there was no data found and removing a big block of indented code.